alarmpy/alarmpy.py

```diff
@@ -25,19 +26,29 @@ def __init__(
         alarm_id=False,
         repeat_alarms=False,
         quiet=False,
+        mqtt_server="",
+        mqtt_port=1883,
+        mqtt_topic="",
     ):
         self.language = language
         self.polling_delay = polling_delay
         self.last_routine_delay = routine_delay
         self.alarm_id = alarm_id
         self.repeat_alarms = repeat_alarms
         self.quiet = quiet
+        self.mqtt_server = mqtt_server
+        self.mqtt_port = mqtt_port
+        self.mqtt_topic = mqtt_topic
 
         self.current_alarms = []
         self.last_routine_output = 0
 
         self.session = self.init_session()
         self.labels = self.load_labels()
+        self.mqtt = mqtt.Client("alamrPyClient")
```

Comment:
Is this hard-coded client name by design?

Reply:
I didn't want to clutter the config. Will fix.